### DISCUSSION TITLE
Node based editor and engine integration?

### DISCUSSION THREAD

**Discussion by amerkoleci** [AvaloniaUI/Avalonia#7239] [Q&A]
Hi guys,
I would love to use Avalonia for my game engine editor, anyone can share, so we can keep track of libraries/snippets about game engine usage, are there examples of d3d11/opengl/vulkan embedding/integration?

Node based components with zoom/pan?

Thanks

**Comment by wieslawsoltes** [comment]
Node editor control https://github.com/wieslawsoltes/NodeEditor
